File: src/FirstBundle/Entity/Child.php
<?php

namespace FirstBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Child
 *
 * @ORM\Table(name="child")
 * @ORM\Entity
 */
class Child
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="lastname", type="string", length=255, nullable=false)
     */
    private $lastname;

    /**
     * @var string
     *
     * @ORM\Column(name="firstname", type="string", length=255, nullable=false)
     */
    private $firstname;

    /**
     * @var string
     *
     * @ORM\Column(name="level", type="string", length=255, nullable=false)
     */
    private $level;

    /**
     * @var integer
     *
     * @ORM\Column(name="parent_id", type="integer", nullable=true)
     */
    private $parentId = 'NULL';

    /**
     * @var integer
     *
     * @ORM\Column(name="class_id", type="integer", nullable=true)
     */
    private $classId = 'NULL';

    /**
     * @var integer
     *
     * @ORM\Column(name="save_nurs_id", type="integer", nullable=true)
     */
    private $saveNursId = 'NULL';

    /**
     * @var integer
     *
     * @ORM\Column(name="save_resto_id", type="integer", nullable=true)
     */
    private $saveRestoId = 'NULL';

    /**
     * @var integer
     *
     * @ORM\Column(name="age", type="integer", nullable=false)
     */
    private $age;

    /**
     * @var string
     *
     * @ORM\Column(name="photo", type="string", length=255, nullable=false)
     */
    private $photo;

    /**
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param int $id
     */
    public function setId($id)
    {
        $this->id = $id;
    }

    /**
     * @return string
     */
    public function getLastname()
    {
        return $this->lastname;
    }

    /**
     * @param string $lastname
     */
    public function setLastname($lastname)
    {
        $this->lastname = $lastname;
    }

    /**
     * @return string
     */
    public function getFirstname()
    {
        return $this->firstname;
    }

    /**
     * @param string $firstname
     */
    public function setFirstname($firstname)
    {
        $this->firstname = $firstname;
    }

    /**
     * @return string
     */
    public function getLevel()
    {
        return $this->level;
    }

    /**
     * @param string $level
     */
    public function setLevel($level)
    {
        $this->level = $level;
    }

    /**
     * @return int
     */
    public function getParentId()
    {
        return $this->parentId;
    }

    /**
     * @param int $parentId
     */
    public function setParentId($parentId)
    {
        $this->parentId = $parentId;
    }

    /**
     * @return int
     */
    public function getClassId()
    {
        return $this->classId;
    }

    /**
     * @param int $classId
     */
    public function setClassId($classId)
    {
        $this->classId = $classId;
    }

    /**
     * @return int
     */
    public function getSaveNursId()
    {
        return $this->saveNursId;
    }

    /**
     * @param int $saveNursId
     */
    public function setSaveNursId($saveNursId)
    {
        $this->saveNursId = $saveNursId;
    }

    /**
     * @return int
     */
    public function getSaveRestoId()
    {
        return $this->saveRestoId;
    }

    /**
     * @param int $saveRestoId
     */
    public function setSaveRestoId($saveRestoId)
    {
        $this->saveRestoId = $saveRestoId;
    }

    /**
     * @return int
     */
    public function getAge()
    {
        return $this->age;
    }

    /**
     * @param int $age
     */
    public function setAge($age)
    {
        $this->age = $age;
    }

    /**
     * @return string
     */
    public function getPhoto()
    {
        return $this->photo;
    }

    /**
     * @param string $photo
     */
    public function setPhoto($photo)
    {
        $this->photo = $photo;
    }

    public function __toString()
    {
return $this->firstname;
    }


}

File: Pimobile/src/com/mycompany/entities/Child.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.entities;

/**
 *
 * @author Ahmed
 */
public class Child {
    private int id;
    private String lastname;
    private String firstname;
    private int level;
    private int parent_id;
    private int class_id;
    private int age;
    private String photo;
    private int teacher_id;

    @Override
    public String toString() {
        return "child{" + "id=" + id + ", lastname=" + lastname + ", firstname=" + firstname + ", level=" + level + ", parent_id=" + parent_id + ", class_id=" + class_id + ", age=" + age + ", photo=" + photo + ", teacher_id=" + teacher_id + '}';
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getLastname() {
        return lastname;
    }

    public void setLastname(String lastname) {
        this.lastname = lastname;
    }

    public String getFirstname() {
        return firstname;
    }

    public void setFirstname(String firstname) {
        this.firstname = firstname;
    }

    public int getLevel() {
        return level;
    }

    public void setLevel(int level) {
        this.level = level;
    }

    public int getParent_id() {
        return parent_id;
    }

    public void setParent_id(int parent_id) {
        this.parent_id = parent_id;
    }

    public int getClass_id() {
        return class_id;
    }

    public void setClass_id(int class_id) {
        this.class_id = class_id;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }

    public String getPhoto() {
        return photo;
    }

    public void setPhoto(String photo) {
        this.photo = photo;
    }

    public int getTeacher_id() {
        return teacher_id;
    }

    public void setTeacher_id(int teacher_id) {
        this.teacher_id = teacher_id;
    }

    public Child() {
    }

    public Child(int id, String lastname, String firstname, int level, int parent_id, int class_id, int age, String photo, int teacher_id) {
        this.id = id;
        this.lastname = lastname;
        this.firstname = firstname;
        this.level = level;
        this.parent_id = parent_id;
        this.class_id = class_id;
        this.age = age;
        this.photo = photo;
        this.teacher_id = teacher_id;
    }

}

